docs/docs/seo-head.md

```diff
@@ -64,40 +63,50 @@ const AboutPage = () => {
 +     </Head>
 ```
 
-You can include any valid `<head>` tag in here that you like, but just to make things easier we also have a utility component [MetaTags](#setting-meta-tags-open-graph-directives).
+You can include any valid `<head>` tag in here that you like. However, Redwood also provides we also have a utility component [&lt;Metadata&gt;](#setting-meta-tags-and-opengraph-directives-with-metadata).
+
+:::caution `<MetaTags>` Deprecation
+
+Prior to Redwood 7.0 this component was called `<MetaTags>` and had several special hard-coded props like `ogContentUrl`, which didn't properly map to the OpenGraph spec. We'll still render `<MetaTags>` for the foreseeable future, but it is deprecated and should migrate to `<Metadata>` if you have an existing app.
+
+:::
+
+### What About Nested Tags?
 
-### What about nested tags?
 Redwood uses [react-helmet-async](https://github.com/staylor/react-helmet-async) underneath, which will use the tags furthest down your component tree.
 
 For example, if you set title in your Layout, and a title in your Page, it'll render the one in Page - this way you can override the tags you wish, while sharing the tags defined in Layout.
 
+:::info Bots & `<meta>` Tags
+
+For these headers to appear to bots and scrapers e.g. for twitter to show your title, you have to make sure your page is prerendered. If your content is static you can use Redwood's built in [Prerender](prerender.md). For dynamic tags, check the [Dynamic head tags](#dynamic-tags)
 
-> **Side note**
-> for these headers to appear to bots and scrapers e.g. for twitter to show your title, you have to make sure your page is prerendered
-> If your content is static you can use Redwood's built in [Prerender](prerender.md). For dynamic tags, check the [Dynamic head tags](#dynamic-tags)
+:::
 
-## Setting meta tags / open graph directives
-Often we want to set more than just the title - most commonly to set "og" headers. Og standing for
-[open graph](https://ogp.me/) of course.
+## Setting `<meta>` Tags and OpenGraph Directives with `<Metadata>`
```

Comment:
Somewhere in this doc, should we also mention that you can just manually write meta tags if you wanted to too with `Head` imported from rwjs/web? 

Sometimes its just easier to copy paste stuff :)

Reply:
Yeah I thought I had a note about that in there somewhere, lemmie check again...

Reply:
The code samples show writing out a plain `<meta>` tag as a child, but I added another sentence that really spells out how to do it.